java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -728,4 +729,85 @@ CompletableFuture<Long> zrangestore(
      * }</pre>
      */
     CompletableFuture<Long> zlexcount(String key, LexRange minLex, LexRange maxLex);
+
+    /**
+     * Returns the union of members from sorted sets specified by the given <code>keys</code>.<br>
+     * In cluster mode, all <code>keys</code> should be mapped to the same <code>hash slot</code>.<br>
```

Comment:
As discussed with @acarbonetto , prefix this sentense with `@apiNote` and put before `@see` section

Reply:
and update other functions too